packages/live-status-gateway/api/schemas/adLibs.yaml

```diff
@@ -37,6 +37,9 @@ $defs:
       id:
         description: Unique id of the AdLib
         type: string
+      partId:
+        description: Unique id of the part which owns this AdLib
+        type: string
```

Comment:
Based on the implementation, it seems that `partId` property will not be present on all `adLib`-type objects, but only on the non-global ones. So while before we just had one `adLib`-type object, we now now have a global adlib type (without `partId`) and another one for adlibs coming from Parts (with `partId`).

I think this should be reflected here in the API, by maybe defining an extension of the `adLib` that includes a required `partId`?

Reply:
+1 for splitting it into two types, one deriving from the other, but do the types in corelib allow us to say that `partId` will always be present on non-global adlibs? I think it's this property, and it's optional https://github.com/nrkno/sofie-core/blob/release51/packages/corelib/src/dataModel/AdLibPiece.ts#L12

Reply:
@jstarpl I've split the typings out a bit more neatly now if that suits?

@ianshade Interestingly making it optional there seems to break an assortment of other bits in the job worker and blueprints integration but I can't figure out why it was marked as optional unless it's possible for an AdLib to be orphaned? So I haven't changed it for the moment.